modules/op-about-pipelines.adoc

```diff
@@ -36,32 +36,44 @@ spec: <4>
   tasks: <7>
   - name: fetch-repository
     taskRef:
-      name: git-clone
-      kind: ClusterTask
+      resolver: cluster
+      params:
+      - name: kind
+        value: task
+      - name: name
+        value: git-clone
+      - name: namespace
+        value: openshift-pipelines
     workspaces:
     - name: output
       workspace: shared-workspace
     params:
-    - name: url
+    - name: URL
       value: $(params.git-url)
-    - name: subdirectory
+    - name: SUBDIRECTORY
       value: ""
-    - name: deleteExisting
+    - name: DELETE_EXISTING
       value: "true"
-    - name: revision
+    - name: REVISION
       value: $(params.git-revision)
   - name: build-image <8>
     taskRef:
-      name: buildah
-      kind: ClusterTask
+      resolver: cluster
+    params:
```

Comment:
wrong indentation - params needs to be at the same level as `resolver` key

Reply:
thanks, fixed